### PR TITLE
Adds LogEntry and LogEntryComment with full text searching

### PR DESCRIPTION
**Pull request by lukecampbell**

### CODE REVIEW

**Comment by DanielJMaher** [comment]
@lukecampbell 3 Tests failed.  Need to resolve ASAP.

**Comment by lukecampbell** [comment]
I will punch you straight in the gaba

**Comment by DanielJMaher** [comment]
As long as the tests start passing, I do not care where you punch me.

File: ooiservices/tests/test_log_entries.py [hunk 1]
```diff
@@ -0,0 +1,285 @@
...
+
+        # GET /log_entry_comment?log_entry_id=:id
+        response = self.client.get('/log_entry_comment?log_entry_id=%s' % entry_id)
+        self.assertEquals(response.status_code, 200)
+        data = json.loads(response.data)
+        self.assertEquals(data['log_entry_comments'], [])
+
+        # POST /log_entry_comment
+        data = {
+            'comment' : u'これもいつか過ぎ去るものです',
```
Comment (lukecampbell) [inline ooiservices/tests/test_log_entries.py]: Boom unicode support

File: ooiservices/tests/test_log_entries.py [hunk 2]
```diff
@@ -0,0 +1,285 @@
...
+                'entry_description' : "It's the end of the world and we know it."
+            }
+        ]
+        # POST /log_entry
+        for entry in entries:
+            response = self.client.post('/log_entry', data=json.dumps(entry), headers=headers)
+            self.assertEquals(response.status_code, 200)
+
+        # GET /log_entry?search=Japanese
+        response = self.client.get('/log_entry?search=Japanese')
```
Comment (lukecampbell) [inline ooiservices/tests/test_log_entries.py]: Full text search

**Comment by DanielJMaher** [comment]
I Like it.

**Pull request merged**